BeeTracker/app/src/main/res/layout/fact_view.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/factView"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:paddingBottom="8dp"
+    >
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/factViewContainer"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:background="#fff">
+
+        <LinearLayout
+            android:id="@+id/factTopBar"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="horizontal"
+            >
+            <TextView
+                android:id="@+id/lblFactId"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textSize="14dp"
+                android:textStyle="bold"
+                android:paddingLeft="16dp"
+                android:paddingTop="16dp"
+                android:paddingBottom="16dp"
+                android:text="TIP 1"
+                android:layout_weight="1"
+                android:textColor="#777"/>
+            <LinearLayout
+                android:layout_width="wrap_content"
+                android:id="@+id/btnClose"
+                android:layout_height="wrap_content"
+                android:paddingLeft="16dp"
+                android:paddingRight="16dp"
+                android:paddingTop="16dp"
+                android:paddingBottom="16dp"
+                android:background="#fff">
+                <ImageView
+                    android:layout_width="16dp"
+                    android:layout_height="16dp"
+                    android:src="@drawable/icon_close"/>
+            </LinearLayout>
+        </LinearLayout>
+        <ImageView
+            android:id="@+id/imgFactIllustration"
+            android:scaleType="centerCrop"
+            android:layout_width="match_parent"
+            android:layout_height="117dp"
```

Comment:
This height is very specific. Why is it such a height? If you just put wrap_content, won't the image just fill itself according to its width?